thirdparty/openapi3/components/schemas/ThirdpartyRequestsAuthorizationsPostRequest.yaml

```diff
@@ -10,17 +10,20 @@ properties:
     type: string
     description: The challenge that the PISP's client is to sign
   transferAmount:
-    $ref: ./Money.yaml
+    allOf:
```

Comment:
Yea... openapi3 validators really don't want `$ref` to have siblings but it's all good and dandy if they are wrapped in `allOF`. We might want to do this for all of the schema to preserve descriptions. I had removed them in the past.

Or...wait for oa3.1 which I believe will allow ref siblings.